### ISSUE TITLE
SQL Script to Create the Database

### ISSUE THREAD

**Issue by TawfeeqBaloush** [HH95/Open-Source-Python-POS-and-Accounting-Software#3]
When I run the program I got the following error
(1049, "Unknown database 'salmanwholeseller'")

It seams the I have to create the database in MySQL first. So my question is how to create this database. Do you have a SQL script to create the database

**Comment by alitheDev** [comment]
> When I run the program I got the following error
> (1049, "Unknown database 'salmanwholeseller'")
> 
> It seams the I have to create the database in MySQL first. So my question is how to create this database. Do you have a SQL script to create the database

yes you have to create database, and the script is available online. u can use any of them.
